clouditor-engine-core/src/main/java/io/clouditor/credentials/CloudAccount.java

```diff
@@ -33,23 +33,34 @@
 import io.clouditor.util.Collection;
 import io.clouditor.util.PersistentObject;
 import java.io.IOException;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Table;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+@Entity(name = "cloud_account")
+@Table(name = "cloud_account")
 @JsonTypeInfo(use = Id.NAME, property = "provider")
 @Collection("accounts")
-public abstract class CloudAccount<T> implements PersistentObject {
+public abstract class CloudAccount<T> implements PersistentObject<String> {
 
   protected static final Logger LOGGER = LoggerFactory.getLogger(CloudAccount.class);
+  private static final long serialVersionUID = 7868522749211998981L;
 
+  @javax.persistence.Id
+  @Column(name = "account_id")
   protected String accountId;
-  protected String user;
+
+  @Column(name = "user")
+  protected String credential;
```

Comment:
Could we keep it as `user` for now? It would be appreciated, because otherwise we need to change the UI code as well - which I would not like to touch in this PR.